config/dws/nnf-ruleset.yaml

```diff
@@ -92,3 +92,18 @@ spec:
         type: "string"
         isRequired: true
         isValueRequired: true
+  - command: "container"
+    watchStates: Proposal,PreRun,PostRun,Teardown
+    ruleDefs:
+      - key: "name"
```

Comment:
This and the "profile" key below require the regexp `'^name$'` and `'^profile$'`, respectively